src/pages/LoginPage.js

```diff
@@ -2,8 +2,38 @@ import React, { Component } from 'react'
 import LoginForm from '../components/LoginForm';
 import '../components/Login.css';
 import '../components/baseStyle.css';
+import { useNavigate } from 'react-router-dom';
+import LoginRequest from '../services/LoginRequest';
 
-export default class LoginPage extends Component {
+class LoginPage extends Component {
+
+    constructor(props){
+        super(props);
+
+        this.state = {
+            error: ''
+        }
+
+        this.logUserIn = this.logUserIn.bind();
```

Comment:
You don't need to bind it, it's an arrow function it by default should have class context.
```suggestion
```